src/segment-loader.js

```diff
@@ -161,6 +151,10 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     // ...for determining the fetch location
     this.fetchAtBuffer_ = false;
+
+    if (settings.debug) {
+      this.logger_ = videojs.log.bind(videojs, 'segment-loader', this.loaderType_, '->');
```

Comment:
If `settings.debug` is `false`, won't all the calls to `this.logger_` throw errors since `this.logger_` will be `undefined`?